solution/ui/regulations/js/src/components/SupplementalContent.vue

```diff
@@ -192,32 +192,41 @@ export default {
     created() {
         let location = ""
         if (window.location.hash) {
-            let section = window.location.hash.substring(1).replace("-", ".");
-            if (section.includes("-")) {
-                // eslint-prefer-destructuring, kinda cool
-                [section] = section.split("-");
-            }
-            if (isNaN(section)) {
-                location = `locations=${this.title}.${this.part}.${section}`
-            }
-            else {
-                location = `locations=${this.title}.${section}`
-                this.selectedPart = `§ ${section}`;
-        }
-        } else {
+          location = this.parseHash(window.location.hash)
+         } else {
             this.fetch_content(this.title, this.part);
         }
         this.fetch_content(this.title, this.part, location)
+        window.addEventListener('hashchange', (event) => {
+          location = this.parseHash(window.location.hash)
```

Comment:
the second argument of `addEventListener` needs to be a named function instance or method so the listener can be removed in the `destroyed` lifecycle.  See explanation here: 

https://deepscan.io/docs/rules/bad-remove-event-listener

It's likely that we'll have to create another method that calls `this.parseHash` and then calls `this.fetch_content` and then use that exact method instance here, as well as in the `removeEventListener` method.